projects/plugins/jetpack/modules/custom-css/custom-css.php

```diff
@@ -30,6 +30,7 @@ public static function add_hooks() {
 		add_action( 'load-revision.php', array( __CLASS__, 'load_revision_php' ) );
 
 		add_action( 'wp_enqueue_scripts', array( __CLASS__, 'wp_enqueue_scripts' ) );
+		add_action( 'admin_footer', array( __CLASS__, 'update_initial_state' ) );
```

Comment:
I don't know if that's the most appropriate hook to use. Let me know if there's a more suited one.